.travis.yml

```diff
@@ -10,6 +10,9 @@ env:
   - DJANGO=1.8 DRF=3.2
   - DJANGO=1.9 DRF=3.2
   - DJANGO=1.9 DRF=3.3
+  - DJANGO=1.9 DRF=3.4
+  - DJANGO=1.10 DRF=3.4
+  - DJANGO=1.10 DRF=3.5
```

Comment:
As described in the other comments, I think we can just get rid of the older version's support since this project is intended for new projects.

Let's just do Django >=1.9 and DRF 3.5